packages/babel-plugin-transform-es2015-shorthand-properties/README.md

```diff
@@ -30,7 +30,7 @@ var cat = {
 
 ```js
 var cat = {
-  getName: function () {
+  getName: function getName() {
```

Comment:
Actually, the current example is correct... the REPL currently only lets you select presets, and not individual plugins... the function is getting its name from `transform-es2015-function-name`.

(p.s. we'd love for someone to help us augment the REPL to fix this!)